R/workhorse.R

```diff
@@ -1,78 +1,145 @@
-build_request <- function(endpoint = NULL,
+build_request <- function(path = NULL,
```

Comment:
we need it now for the 2 endpoints that don't have unique paths (`.drive.files.create` and `.drive.files.update`) we could break from the Google API docs and make up our own endpoints to make them unique if you think that'd be better